brave/brave-core
# Updating welcome page text for anon wallets

Fixes: https://github.com/brave/brave-browser/issues/6602

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Open browser
2. Wait for initialization
3. Close browser
4. Open Preferences file and search for countryid_at_install
5. Replace current value with 19024
6. Restart Browser
7. Navigate to `brave://rewards`
8. Confirm updated text is shown on welcome page

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
